File: tests/indexing/parsing/lang/fixtures/swift/Conditionals.swift
func classify(score: Int, bonus: Int) -> String {
    if score >= 90 {
        let total = score + bonus
        return "top \(total)"
    }

    if score >= 50 {
        return "pass"
    } else if score >= 30 {
        let gap = 50 - score
        return "near \(gap)"
    } else {
        return "fail"
    }
}

func firstPositive(values: [Int]) -> Int {
    guard let head = values.first else {
        let fallback = values.count
        return -fallback
    }

    if head > 0 {
        return head
    }
    return -head
}

func describe(value: Int?) -> String {
    if let value = value {
        let doubled = value * 2
        return "some \(doubled)"
    } else {
        return "none"
    }
}

File: tests/indexing/parsing/lang/fixtures/c/Conditionals.c
int classify(int n) {
    int score = 0;
    if (n < 0) {
        score = -n;
        score += 1;
        return score;
    }
    if (n == 0) {
        score = 100;
        score /= 2;
    } else {
        score = n * 3;
        score -= 4;
    }
    return score;
}

const char *grade(int marks, int bonus) {
    if (marks + bonus >= 90) {
        int total = marks + bonus;
        return total > 95 ? "A+" : "A";
    } else if (marks >= 50) {
        int deficit = 70 - marks;
        return deficit > 0 ? "C" : "B";
    } else {
        return "F";
    }
}
